### DISCUSSION TITLE
Support filtering by compound extension files

### DISCUSSION THREAD

**Discussion by cody-elhard** [community/community#55644] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This is not a move the needle project, but a common use case for us using GraphQL is to filter out files that are auto generated to get a better idea of the actual human code that went into the project.

This would make for a better review experience and span across a lot more domains than GraphQL.

**Comment by look** [comment]
Can you give an example of the type of search and results you are expecting? If you can link to a public repository with the type of files you're talking about that would be helpful.

In case you are looking to exclude files path path, that's possible. You can use a regular expression: `NOT path:/\.some\.extension$/`
